internal/components/components.go

```diff
@@ -58,6 +58,28 @@ func FindComponentsByType(t *testing.T, dut *ondatra.DUTDevice, cType oc.E_Platf
 	return s
 }
 
+// FindSWComponentsByType finds the list of SW components based on a type.
+func FindSWComponentsByType(t *testing.T, dut *ondatra.DUTDevice, cType oc.E_PlatformTypes_OPENCONFIG_SOFTWARE_COMPONENT) []string {
+	components := gnmi.GetAll[*oc.Component](t, dut, gnmi.OC().ComponentAny().State())
+	var s []string
+	for _, c := range components {
+		if c.GetType() == nil {
+			t.Logf("Component %s type is missing from telemetry", c.GetName())
```

Comment:
these logging is not very helpful and fill up the test log, can u please remove that